test/integration/controller/core/suite_test.go

```diff
@@ -75,7 +75,7 @@ func managerSetup(mgr manager.Manager, ctx context.Context) {
 	controllersCfg := &config.Configuration{}
 	controllersCfg.Metrics.EnableClusterQueueResources = true
 	controllersCfg.QueueVisibility = &config.QueueVisibility{
-		UpdateIntervalSeconds: 1,
+		UpdateIntervalSeconds: 2,
```

Comment:
why change this?

Reply:
Bumped the update interval to 2s to make the test fail almost every time on master, while currently it passes almost every time. Added a note in the PR description.